format/midi/constants.go

```diff
@@ -368,9 +370,9 @@ var framerates = scalar.UintMapSymUint{
 	3: 30,
 }
 
-var fps = scalar.UintMapSymUint{
-	0xe8: 24,
-	0xe7: 25,
-	0xe6: 29,
-	0xe5: 30,
+var fps = scalar.SintMapSymUint{
```

Comment:
I noticed above `manufacturers_extended`, maybe use camel case to be more like reste of go code? maybe there are others?

I've usually named these mappers `<thing>Names`, `<thing>Map` etc maybe good here also?

Reply:
Makes sense .. renamed throughout.